collector.go

```diff
@@ -59,6 +59,24 @@ var (
 		"Atmospheric pressure measurement in mb",
 		varLabels,
 		nil)
+
+  windDesc = prometheus.NewDesc(
+    prefix+"wind_kmh",
+    "Wind strength in kmh",
+    varLabels,
+    nil)
+
+  windangleDesc = prometheus.NewDesc(
+    prefix+"wind_angle",
+    "Wind angle in degrees",
+    varLabels,
+    nil)
+
+  rainDesc = prometheus.NewDesc(
+    prefix+"rain_mm",
```

Comment:
So it is already pre-aggregated in some kind of way, I think we should mention that in the description possibly also in the name.